src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml

```diff
@@ -7,11 +7,13 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:sampleData="clr-namespace:GitHub.SampleData;assembly=GitHub.App"
              xmlns:ui="clr-namespace:GitHub.UI;assembly=GitHub.UI"
+             xmlns:automation="clr-namespace:GitHub.VisualStudio.TestAutomation;assembly=GitHub.VisualStudio.TestAutomation"
              Background="Beige"
              d:DataContext="{d:DesignInstance Type=sampleData:InfoPanelDesigner, IsDesignTimeCreatable=True}"
              d:DesignHeight="300"
              d:DesignWidth="300"
-             mc:Ignorable="d">
+             mc:Ignorable="d"
+             AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.GitHubToolBar}">
```

Comment:
Yeah you're right, I think I made a new ID for this but its not reflected in here somehow.